parachain/runtime/interlay/src/lib.rs

```diff
@@ -1263,7 +1265,7 @@ construct_runtime! {
         // # Lending & AMM
         Loans: loans::{Pallet, Call, Storage, Event<T>, Config} = 100,
         DexGeneral: dex_general::{Pallet, Call, Storage, Event<T>} = 101,
-        DexStable: dex_stable::{Pallet, Call, Storage, Event<T>}  = 102,
+        DexStable: dex_stable::{Pallet, Call, Storage, Event<T>}  = 102, // note: calls disabled
```

Comment:
I think you forgot to remove `Call`?

Reply:
I see it was removed from the Kintsugi runtime, but anyway I'm not sure if we should remove since then it would require another runtime upgrade if we decide to add a normal stable pool (non-rebasing). Let's confirm with @nud3l or @philippb90.

Reply:
I'd like to keep the stable swap. If all goes well, we can get wBTC sometime in late July and then a iBTC/wBTC pool would definitely be one of the first things, we'd want to add.

Reply:
I assumed we'd need changes anyway before we start using it. At the very least we'd need proper benchmarks

Reply:
No changes would be required for adding a iBTC/wBTC pool since the price is the same, I'm not sure what the plan is regarding rebasing token support. As discussed separately let's keep the calls enabled and I will review the missing benchmarks once this PR is merged.